supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/request/GetRequestHandler.scala

```diff
@@ -78,7 +79,9 @@ trait GetRequestHandler { this: RequestHandler =>
       }
     } catch {
       case npe: NullPointerException =>
-        throw new RuntimeException(s"Caught NPE while processing operation ${operation.getName}: $operation", npe)
+        logger.error(s"Caught NPE while interpreting operation ${operation.getName} from the following JSON response: $operation")
+
+        throw npe
```

Comment:
TOL logging *and* throwing is usually considered an antipattern as it usually results in multiple copies of the same error message, though it sounds like we were suffering from the opposite problem here. Though it looks like this may be intentionally omitting the NPE from `logger.error(...)` to mitigate that issue somewhat?

Reply:
We want both the JSON and the stack trace to make it into the log.

Unfortunately, I could not find a way to feed the stack trace of `npe` into `new RuntimeException()` such that it would get printed out, so we still need `throw npe`.

Meanwhile, there is no way to add a custom message to `npe`, hence the log statement.

I agree it's a bit goofy but I did not want to invest too much time in code that is a candidate for rewriting.